<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeatingControlPage</class>
 <widget class="QWidget" name="HeatingControlPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>457</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#titleLabel {
	padding-width-left:5px;
	padding-width-right:5px;
	border: 1px solid white;
	border-radius: 10px;
	min-width: 200px;
	min-height: 40px;
}

QLabel#statusLabel {
	background-color: white;
}

 QComboBox {
	selection-background-color: transparent;
	selection-color: black;
}

 QGroupBox {
     border: 1px solid gray;
     border-radius: 5px;
 }
QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top left; /* position at the top center */
 }

QFrame#frame_1,#frame_2  {
	border: 2px solid rgb(125, 125, 125, 50%);
	border-radius: 3px;
	background-color:rgb(100, 200, 200, 50%);
}

QFrame#frame_1 QPushButton {
	min-width: 100px;
	min-height: 100px;
	max-width: 100px;
	max-height: 100px;
	border-image: url(:/Hac/round-button.png);
}

QFrame#frame_2 QPushButton {
	min-width: 55px;
	min-height: 55px;
	max-width: 55px;
	max-height: 55px;
	border-image: url(:/Hac/round-button-small.png);
}

 QPushButton#scheduleButton {
	image: url(&quot;:/Hac/clock.png&quot;);
 }

QPushButton#tempUpButton {
	image: url(:/Hac/up-arrow-black.png);
}
QPushButton#tempUpButton:pressed {
	image: url(:/Hac/up-arrow-red.png);
}
QPushButton#tempDownButton {
	image: url(:/Hac/down-arrow-black.png);
}
QPushButton#tempDownButton:pressed {
	image: url(:/Hac/down-arrow-red.png);
}

QPushButton#blowUpButton {
	image: url(:/Hac/up-arrow-black.png);
}
QPushButton#blowUpButton:pressed {
	image: url(:/Hac/up-arrow-red.png);
}
QPushButton#blowDownButton {
	image: url(:/Hac/down-arrow-black.png);
}
QPushButton#blowDownButton:pressed {
	image: url(:/Hac/down-arrow-red.png);
}

QPushButton#startButton {
	image: url(:/Hac/right-arrow-black.png);
}
QPushButton#startButton:pressed {
	image: url(:/Hac/right-arrow-red.png);
}
QPushButton#stopButton {
	image: url(:/Hac/stop.png);
}
QPushButton#stopButton:pressed {
	image: url(:/Hac/stop-red.png);
}

QSpinBox {
	margin: 1px;
	selection-background-color: transparent;
	text: black;
	selection-color: black;
	font: normal 15px;
	border: 1px solid black;
	padding-width-right: 40px;
}

QSpinBox::up-button {
	subcontrol-origin: border;
    subcontrol-position: right center;
	width: 20px;
	height: 20px;
}

QSpinBox:up-arrow {
     image: url(:/Hac/up-arrow-black.png);
     width: 20px;
     height: 20px;
}
QSpinBox::up-arrow:pressed {
     image: url(:/Hac/up-arrow-red.png);
}


QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: right center;
	left: -20px;
	width: 20px;
	height: 20px;
}

QSpinBox::down-arrow {
     image: url(:/Hac/down-arrow-black.png);
     width: 20px;
     height: 20px;
}
QSpinBox::down-arrow:pressed {
     image: url(:/Hac/down-arrow-red.png);
}


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Heating Control</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>140</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_1">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="startButton"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="stopButton"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="scheduleButton"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="margin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Heating Mode</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_2">
           <property name="font">
            <font>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Business</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Comfortable</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Economical</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Programming</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Temperature</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="3" column="1" colspan="2">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Set To</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="tempSetLabel">
             <property name="font">
              <font>
               <pointsize>25</pointsize>
              </font>
             </property>
             <property name="text">
              <string>27</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="2" colspan="2">
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <family>unifont</family>
               <pointsize>25</pointsize>
              </font>
             </property>
             <property name="text">
              <string>℃</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>unifont</family>
               <pointsize>25</pointsize>
              </font>
             </property>
             <property name="text">
              <string>℃</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="currentLabel">
             <property name="font">
              <font>
               <pointsize>25</pointsize>
              </font>
             </property>
             <property name="text">
              <string>25</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="tempUpButton">
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tempDownButton">
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/hac.qrc"/>
 </resources>
 <connections/>
</ui>
